src/views/EditorHome.vue

```diff
@@ -23,7 +23,7 @@
               }}
             </div>
 
-            <template v-if="isEngineWaitingLong">
+            <template v-if="isEngineWaitingLong && isMultipleEngine">
               <q-separator spaced />
               エンジン起動に時間がかかっています。<br />
               <q-btn outline @click="restartAppWithSafeMode">
```

Comment:
「エンジン起動に時間がかかっています」を表示するのは別に良いと思うんですよね〜。
なのでこうするのとかどうでしょう。

```suggestion
            <template v-if="isEngineWaitingLong">
              <q-separator spaced />
              エンジン起動に時間がかかっています。<br />
              <q-btn v-if="isMultipleEngine" outline @click="restartAppWithSafeMode">
```

まあでも表示されたところでユーザーができること変わらないし、別に今のままでも良いかもです。
どちらでも！

Reply:
別に何もできないので出す必要もないかなーと思います。
トラブルシューティング的なページがあればそこに飛ばすリンクを貼るのもありかも？

Reply:
あーーー　それはかなり良いですね！！
リンクはQ&Aページが良いかも。 https://voicevox.hiroshiba.jp/qa

大体PC再起動でなおるので（前のエンジンが起動しっぱなしとかが多いっぽい？）それQ&Aに追記しておくので、とりあえずQ&A開くボタン表示するとかとても良さそう。

Reply:
出すようにしてみました。